### PR TITLE
UI highlighting sections 1-4

### PR DESCRIPTION
**Pull request by quickcharge3**
tango, scans, glyph, neutral items and backpack, sentry ward. minimap is not working as hoped

### CODE REVIEW

File: game/scripts/vscripts/Sections/Chapter1/SectionShopUI.ts [hunk 1]
```diff
@@ -27,8 +27,10 @@ const onStart = (complete: () => void) => {
     const playerHero = getPlayerHero();
     if (!playerHero) error("Could not find the player's hero.");
 
+    // UI Highlighting Paths
     const shopBtnUIPath = "HUDElements/lower_hud/shop_launcher_block/ShopCourierControls/ShopButton"
     const tangoInGuideUIPath = "HUDElements/shop/GuideFlyout/ItemsArea/ItemBuildContainer/ItemBuild/Categories/ItemList/Item44"
+    const InventorySlot_0_UIPath = "HUDElements/lower_hud/center_with_stats/center_block/inventory/inventory_items/InventoryContainer/inventory_list_container/inventory_list/inventory_slot_0"
```
Comment (RobinKa) [inline game/scripts/vscripts/Sections/Chapter1/SectionShopUI.ts:33]: please follow our (=the usual typescript) conventions, no underscores and camelCase

File: game/scripts/vscripts/Sections/Chapter2/SectionTower.ts [hunk 2]
```diff
@@ -59,6 +59,9 @@ const onStart = (complete: () => void) => {
     // Get or create the Dire T1 tower
     const direTopTower = getDireTopTower();
 
+    // UI Highlighting Paths
+    const GlyphUIPath = "HUDElements/minimap_container/GlyphScanContainer/glyph/NormalRoot/GlyphButton"
```
Comment (RobinKa) [inline game/scripts/vscripts/Sections/Chapter2/SectionTower.ts:63]: +

File: game/scripts/vscripts/Sections/Chapter3/Chapter3.ts [hunk 3]
```diff
@@ -16,6 +16,11 @@ const requiredState: RequiredState = {
     requireSunsfanGolem: true,
 }
 
+// UI Highlighting Paths
+const NeutralSlotUIPath = "HUDElements/lower_hud/center_with_stats/inventory_composition_layer_container/inventory_neutral_slot_container/inventory_neutral_slot"
+const InventorySlot_6_UIPath = "HUDElements/lower_hud/center_with_stats/center_block/inventory/inventory_items/InventoryContainer/inventory_backpack_list/inventory_slot_6/ButtonAndLevel"
+const InventorySlot_7_UIPath = "HUDElements/lower_hud/center_with_stats/center_block/inventory/inventory_items/InventoryContainer/inventory_backpack_list/inventory_slot_7/ButtonAndLevel"
```
Comment (RobinKa) [inline game/scripts/vscripts/Sections/Chapter3/Chapter3.ts:22]: +

File: game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts [hunk 4]
```diff
@@ -60,6 +65,10 @@ function onStart(complete: () => void) {
                 }
             }),
             tg.immediate(_ => canPlayerIssueOrders = false),
+            tg.immediate(_ => {
+                // highlightUiElement(MinimapUIPath, 5, true)
+                // print("crappy minimap highlight on line 70");
+            }),
```
Comment (RobinKa) [inline game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts:71]: Remove this

File: game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts [hunk 5]
```diff
@@ -30,9 +30,14 @@ const scanDuration = 8;
 const radiantCreepsNames = [CustomNpcKeys.RadiantMeleeCreep, CustomNpcKeys.RadiantMeleeCreep, CustomNpcKeys.RadiantMeleeCreep, CustomNpcKeys.RadiantMeleeCreep, CustomNpcKeys.RadiantRangedCreep];
 let radiantCreeps: CDOTA_BaseNPC[] = [];
 
+// UI Highlighting Paths
+const ScanUIPath = "HUDElements/minimap_container/GlyphScanContainer/RadarButton/NormalRoot/RadarIcon"
+const MinimapUIPath = "HUDElements/minimap_container/HUDSkinMinimap"
```
Comment (RobinKa) [inline game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts:35]: +

File: game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts [hunk 6]
```diff
@@ -19,6 +19,8 @@ const requiredState: RequiredState = {
 const dustName = "item_dust";
 const dustLocation = Vector(-1500, 4000, 256);
 
+const InventorySlot_0_UIPath = "HUDElements/lower_hud/center_with_stats/center_block/inventory/inventory_items/InventoryContainer/inventory_list_container/inventory_list/inventory_slot_0"
```
Comment (RobinKa) [inline game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts:22]: +

**Review by RobinKa — CHANGES_REQUESTED** [review]
Some issues with conventions, need to remove highlights on stop, otherwise looks fine

File: game/scripts/vscripts/Sections/Chapter4/SectionWards.ts [hunk 7]
```diff
@@ -18,6 +18,7 @@ const requiredState: RequiredState = {
 
 const markerLocation = Vector(-2200, 3800, 256);
 const wardLocation = Vector(-3400, 3800);
+const wardLocationSentry = Vector(-3400, 4000);
```
Comment (RobinKa) [inline game/scripts/vscripts/Sections/Chapter4/SectionWards.ts:21]: Probably want to spawn the sentry ward there too (assuming it still uses wardLocation?)

Comment (quickcharge3) [reply game/scripts/vscripts/Sections/Chapter4/SectionWards.ts:21]: it does, will change

File: game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts (hunk 4 above)
Comment (quickcharge3) [reply game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts:71]: will remove this

**Pull request closed without merging**